src/Windows/Avalonia.Win32/WindowImpl.cs

```diff
@@ -331,11 +331,12 @@ public virtual void Show()
             ShowWindow(_showWindowState);
         }
 
-        public void BeginMoveDrag()
+        public void BeginMoveDrag(PointerPressedEventArgs e)
         {
             WindowsMouseDevice.Instance.Capture(null);
             UnmanagedMethods.DefWindowProc(_hwnd, (int)UnmanagedMethods.WindowsMessage.WM_NCLBUTTONDOWN,
                 new IntPtr((int)UnmanagedMethods.HitTestValues.HTCAPTION), IntPtr.Zero);
+            e.Pointer.Capture(null);
```

Comment:
@kekekeks is this correct? im not quite sure

Reply:
It should be correct, but we need to test with an actual touchscreen.

Reply:
is the WindowsMouseDevice call still needed?